datafusion/physical-expr/src/expressions/in_list.rs

```diff
@@ -489,6 +496,196 @@ impl InListExpr {
             contains_null
         ))
     }
+
+    fn evaluate_non_dict(
```

Comment:
This is just a refactor to move all available on into this function so it can be reused with Dictionary data